navigation/ca_move_base/maps/apriltags.yaml

```diff
@@ -0,0 +1,6 @@
+free_thresh: 0.196
```

Comment:
Instead of uploading this you could avoid calling the node `map_server` when `LOCALIZATION=none`.

Reply:
Deleted `apriltags.yaml`